vernac/record.ml

```diff
@@ -37,6 +31,7 @@ module RelDecl = Context.Rel.Declaration
 (** Flag governing use of primitive projections. Disabled by default. *)
 let primitive_flag = ref false
 let () =
+  let open Goptions in
   declare_bool_option
```

Comment:
While you're at it these should be _and_ref

Reply:
Done thanks.